ichnaea/api/exceptions.py

```diff
@@ -152,6 +152,17 @@ class ParseError(BaseAPIClientError):
     domain = "global"
     reason = "parseError"
     message = "Parse Error"
+    error_details = None
+
+    def __init__(self, details=None):
+        self.error_details = details
+        super().__init__()
+
+    def json_body(self):
+        json = super().json_body()
+        if self.error_details:
+            json["details"] = self.error_details
+        return json
```

Comment:
I probably wouldn't use the variable name `json` since if this file ever imports the `json` module, then that gets problematic. However, it looks like other parts of this file use `json` as a variable name, so meh.

Reply:
I agree, it can be problematic. As long as I'm touching it, I should touch them all at once.